engine/src/physics/solver/penetration_constraint/physics_constants_pc.hpp

```diff
@@ -0,0 +1,36 @@
+/// @file
+/// @brief Resource @ref cubos::engine::PhysicsConstantsPC.
+/// @ingroup physics-plugin
+
+#pragma once
+
+#include <cubos/core/reflection/reflect.hpp>
+
+#include <cubos/engine/api.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Resource which allows configuration over magic numbers in the solver plugin.
+    /// @ingroup physics-plugin
+    struct PhysicsConstantsPC
+    {
+        CUBOS_REFLECT;
+
+        float maxBias = -4.0F;
+
+        float minContactHertz = 30.0F;
+
+        float minKNormal = 0.0F;
+
+        float minKFriction = 0.0F;
+
+        float minTangentLenSq = 1e-6F * 1e-6F;
+
+        float minRestitution = 0.0F;
+
+        float minNormalSpeed = -0.01F;
+
+        float minNormalImpulse = 0.0F;
```

Comment:
Same as before, it would be good to have these things documented! Otherwise they won't get picked up by doxygen when generating our doc pages.